web/app/js/components/StatusTable.jsx

```diff
@@ -42,11 +42,7 @@ const columns = {
       dataIndex: "name",
       key: "name",
       render: (name, record) => {
-        if (shouldLink && _.some(record.statuses, ["value", "good"])) {
-          return <Link to={`${pathPrefix}/deployment?deploy=${name}`}>{name}</Link>;
-        } else {
-          return name;
-        }
+        return <Link to={`${pathPrefix}/deployment?deploy=${name}`}>{name}</Link>;
```

Comment:
I think we still need to check the `shouldLink` variable here, since we don't want to link the conduit controller components listed on the service mesh overview page. On your branch it looks like this:

![screen shot 2017-12-15 at 12 17 37 pm](https://user-images.githubusercontent.com/9226/34059053-310440b4-e192-11e7-962f-047d7dd09773.png)

And all of those links 404. If you re-add the shouldLink check, that should fix it.

Reply:
This code change is also causing lint warnings on your branch, due to the fact that you're no longer using the `shouldLink` and `record` variables:

```
  39:18  warning  'shouldLink' is defined but never used. Allowed unused args must match /^_/  no-unused-vars
  44:22  warning  'record' is defined but never used. Allowed unused args must match /^_/      no-unused-vars
```

It's good practice to fix these as they arise, so that we don't have to go back and clean them up later.